lib/seo.php

```diff
@@ -234,6 +234,7 @@ public function sendSitemap($domain = '')
         }
         $sitemap = rex_extension::registerPoint(new rex_extension_point('YREWRITE_SITEMAP', $sitemap));
 
+        ob_clean(); // Workaround aufgrund der Ausgabe von zwei Leerzeichen am Dokumentanfang -> führt zum unglütigem XML Dokument
```

Comment:
wo kommen die 2 leerzeichen her?

meistens sind das irgendwelche leerzeilen die in dateien nach dem php ende tag `?>` stehen